spec/unit/appmgmt_spec.rb

```diff
@@ -46,7 +46,7 @@ def compile_to_env_catalog(string)
 
       application app {
         prod { one: host => ahost, export => Cap[cap] }
-        cons { two: host => ahost, consume => Cap[cap] }
```

Comment:
One is a path without problem (the one removed), the other exposes a problem = different code paths. Line 48 is for export.